homeassistant/components/repetier/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "repetier",
   "name": "Repetier-Server",
   "documentation": "https://www.home-assistant.io/integrations/repetier",
-  "requirements": ["pyrepetier==3.0.5"],
+  "requirements": ["pyrepetierng==0.1.0"],
```

Comment:
Please link to the Github repo of the new library.

Has the old library pyrepetier been removed from Github? I can't find it anymore. Have you been in contact with the maintainer of pyrepetier?

Reply:
Hello. Should I add the new links as a comment? 
Yes, the old pyrepetier got removed by the owner. The developer of the old library was @MTrab himself.
Since the license is MIT, I took the source code from the package fixed the issue and republished it as pyrepetierng

Reply:
https://github.com/ShadowBr0ther/pyrepetier-ng
This is the link to the republished library

Reply:
Please update the PR description with this background info and link to repo and if possible a Github commit compare view of the diff between pyrepetier==3.0.5 and pyrepetierng==0.1.0. Thanks!

Reply:
I updated the description. Do you need any more information?

Reply:
No, that's good. Thanks!

Some recommendations for the library:
- The egg-info directory is normally not committed to source. It's only needed when building the distribution to publish on PyPI.
  - https://github.com/ShadowBr0ther/pyrepetier-ng/tree/main/src/pyrepetierng.egg-info
- I recommend updating the `.gitignore` file with standard Python things to ignore.
  - https://github.com/github/gitignore/blob/main/Python.gitignore
- It would be great if you also tag Github releases corresponding to the PyPI releases, to make it easier for the community to track changes in the library.

Reply:
Will do it. Thanks!
Can this  now get merged?